bioblend/_tests/pytest_galaxy_test_wrapper.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import os
+import sys
+
+try:
+    import pytest
```

Comment:
`setup.py` already has `nose>=1.3.1` in `tests-require`, any specific reason not to use that instead?